lib/class-wp-theme-json.php

```diff
@@ -177,7 +177,7 @@ class WP_Theme_JSON {
 	 */
 	const PRESETS_METADATA = array(
 		array(
-			'path'          => array( 'settings', 'color', 'palette' ),
```

Comment:
I think we already discussed this? I may be dreaming, though :sweat_smile: so I'm going to list why I don't think this is a good idea:

- this makes all the code that processes the presets know where they are
- a reader of this can't know where these are sourced from
- we can't automate updating the docs from this info without also holding a reference to `settings`

Reply:
Hi @nosolosw,
I applied this change here because as part of this PR I needed to retrieve and set presets from deactivatedSettings instead of from settings.
```

						gutenberg_experimental_set(
							$this->contexts[ $context ]['deactivatedSettings'],
							$preset['path'],
```

Reply:
With the caveat that I'm not sure this should work this way, as I mentioned below, I also wanted to share that perhaps is a way to balance both needs?